spacy_llm/backends/rest/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .registry import backend_rest
+from .backend.base import Backend
 
-__all__ = ["backend_rest"]
+__all__ = ["backend_rest", "Backend"]
```

Comment:
Why do we need to re-export `Backend`? By the way, should we rename it, eg to `RESTBackend`?

Reply:
This was only done for more convenient access with the `Generic` approach. Reversed in https://github.com/explosion/spacy-llm/pull/117/commits/61f5984861e954cd570e87d47a3f92eef18ca276.